### ISSUE TITLE
Symfony\Component\Messenger\Exception\HandlerFailedException: Expected response code 250 but got an empty response ---> Swift_TransportException: Expected response code 250 but got an empty response

### ISSUE THREAD

**Issue by gcilinskas** [swiftmailer/swiftmailer#1291]
Symfony\Component\Messenger\Exception\HandlerFailedException: Expected response code 250 but got an empty response ---> Swift_TransportException: Expected response code 250 but got an empty response
at Symfony\Component\Console\Application->run() in /home/narbutas/narbutas/releases/134/bin/console:line 39
--- End of inner exception stack trace ---
at Swift_Mailer->send() in /home/narbutas/narbutas/releases/134/src/Service/Mail/EmailSender.php:line 79
at App\Service\Mail\EmailSender->sendEmail() in /home/narbutas/narbutas/releases/134/src/Service/Mail/EmailSender.php:line 163
at App\Service\Mail\EmailSender->sendEmailWithMandrillTemplate() in /home/narbutas/narbutas/releases/134/src/MessageHandler/OrderStatusEmailHandler.php:line 173
at App\MessageHandler\OrderStatusEmailHandler->handle() in /home/narbutas/narbutas/releases/134/src/MessageHandler/HandlerTrait.php:line 35
at App\MessageHandler\OrderStatusEmailHandler->processMessage() in /home/narbutas/narbutas/releases/134/src/MessageHandler/OrderStatusEmailHandler.php:line 100
at Symfony\Component\Console\Application->run() in /home/narbutas/narbutas/releases/134/bin/console:line 39

| Q                   | A
| ------------------- | -----
| Bug report?         | yes
| Feature request?    | no
| RFC?                | no
| How used?           | Symfony
| Swiftmailer version | 6.1.3
| PHP version         | 7.2

### Flow of the code:
1. Email event is triggered
2. Producer produces Message
3. AWS SQS Receives Message
4. SQS calls Symfony message handler to handle message
5. Mandrill messages/render API call is made to generate an email template
6. SwiftMailer::send method is called

### swiftmailer.yaml config (in current case sync_mailer is used)
```
swiftmailer:
    default_mailer: spooled_mailer
    mailers:
        spooled_mailer:
            url: '%env(MAILER_URL)%'
            spool: { type: 'memory' }
            stream-options:
                ssl:
                    allow_self_signed: true
                    verify_peer: false
        sync_mailer:
            url: '%env(MAILER_URL)%'
            stream-options:
                ssl:
                    allow_self_signed: true
                    verify_peer: false

```
### Mailer URL
MAILER_URL=smtp://smtp.mandrillapp.com:587?timeout=600?encryption=tls&auth_mode=login&username="USERNAMEXX"&password="PASSWORDXX"

### Bug Explanation
In short, the problem is occurring only when emails are sent using AWS SQS. The First 1-2 attempts to send emails usually are successful, but all of the following emails are not sent, and the given error is thrown. 
When executing code without AWS SQS, emails are always sent correctly.

Could you please provide some information about this case? All older similar bugs are closed without a clear explanation
